### ISSUE TITLE
"Windows Media Player cannot play the file because there is a problem with your sound device"

### ISSUE THREAD

**Issue by Tacocat91** [cDima/Aerial#132]
I have downloaded/used the screensaver before on another laptop, but I cannot get it to work on this new one.

Running Windows 10 using Aerials v6.1. I have all updated sound device drivers, no other applications have any issues, I wasn't aware that these videos even had sound.

Re-installed WMP, ran Windows updates, ensured that .NET was updated (v4.6)

![2017-08-08 16_09_56-aerial screensaver](https://user-images.githubusercontent.com/30845785/29094843-67e702a4-7c54-11e7-9edd-7f5e6c64d6f9.png)

**Comment by alistairmcmillan** [comment]
@Tacocat91 So if you download one of Apple's aerial movies and just try to play it in Windows Media Player does it play okay, or do you still get this error?

Here's a link to one of the movies.
http://a1.phobos.apple.com/us/r1000/000/Features/atv/AutumnResources/videos/b1-1.mov

**Comment by Tacocat91** [comment]
Yes, I have gone to the cache location and I am able to play the videos though WMP without any issue, I was able to download and play that video as well.

![2017-08-09 08_28_30-windows media player](https://user-images.githubusercontent.com/30845785/29123891-dbf7a300-7cdc-11e7-8870-b5e3cdd5615e.png)

**Comment by Tacocat91** [comment]
Any movement on this? Just wondering if I can try anything else.

**Comment by shpagaalex** [comment]
Friends, this video showed an error
http://a1.phobos.apple.com/us/r1000/000/Features/atv/AutumnResources/videos/comp_GL_G010_C006_v08_6Mbps.mov

I saw it when I downloaded it to C: \ Users \ Shpagin \ AppData \ Local \ Aerial \ temp
![1](https://user-images.githubusercontent.com/32498436/31149520-426e2510-a8a1-11e7-906b-fc30c6f11dc6.jpg)

**Comment by zan5hin** [comment]
I have this error as well. Just installed fall creators update and the default screensaver (ribbon) is giving me errors, so I discovered Aerial.

**Comment by kamilmirza** [comment]
facing this error again
all cached files are running via Windows Media Player
tried everything - fresh install - removing cache - removing custom codecs 
all other default screensavers are working on my end, on Windows Fall Creators Update

**Comment by zan5hin** [comment]
An update - this is now FIXED on my machine.  I got an alert of a new version of K-Lite codec pack and after install it is now working! 

Version 13.6.5 Full ~ 46.6 MB ~ November 9th 2017
